recipes/folly/all/conanfile.py

```diff
@@ -254,3 +268,30 @@ def package_info(self):
         self.cpp_info.components["libfolly"].names["cmake_find_package_multi"] = "folly"
         self.cpp_info.components["libfolly"].set_property("cmake_target_name", "Folly::folly")
         self.cpp_info.components["libfolly"].set_property("pkg_config_name", "libfolly")
+
+        if Version(self.version) >= "2019.10.21.00":
+            self.cpp_info.components["follybenchmark"].set_property("cmake_target_name", "Folly::follybenchmark")
```

Comment:
Where does this come from 🤔 looking at the latest it's not there https://github.com/facebook/folly/search?q=follybenchmark

🤢 

https://github.com/facebook/folly/blob/8874256376d2f8a32867f17c9472a446d6707604/folly/CMakeLists.txt#L22 


at some point this needs to be removed

Reply:
> Where does this come from 🤔 looking at the latest it's not there https://github.com/facebook/folly/search?q=follybenchmark
> 
> 🤢
> 
> https://github.com/facebook/folly/blob/8874256376d2f8a32867f17c9472a446d6707604/folly/CMakeLists.txt#L22
> 
> at some point this needs to be removed

hi @prince-chrismc  
follybenchmark target is still there : https://github.com/facebook/folly/blob/main/folly/CMakeLists.txt#L23

Reply:
I clearly did not drink enough coffee.

Perfect thank you for pointign that out!